appeals/web/src/server/lib/appellant-case.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
Not sure about an 'appellant-case' lib module. Probably could be generalised, as the invalid reasons are id/name pairs and there will be a lot of those coming from the api, but happy to leave it and maybe just rename it?

Reply:
Well spotted, I think I was too focused on the specific ticket when writing this. I've updated this to a generic helper function which will work with any id/name pairs, and updated the naming accordingly (also added a unit test)